tests/system/action/theme/test_delete.py

```diff
@@ -0,0 +1,34 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class ThemeDeleteActionTest(BaseActionTestCase):
```

Comment:
Could you please add a test, showing that you can't delete the default theme of the organization